site2/docs/deploy-bare-metal.md

```diff
@@ -326,15 +326,15 @@ Flag | Description
 
 [BookKeeper](https://bookkeeper.apache.org) handles all persistent data storage in Pulsar. You need to deploy a cluster of BookKeeper bookies to use Pulsar. You can choose to run a **3-bookie BookKeeper cluster**.
 
-You can configure BookKeeper bookies using the [`conf/bookkeeper.conf`](reference-configuration.md#bookkeeper) configuration file. The most important step in configuring bookies for our purposes here is ensuring that [`zkServers`](reference-configuration.md#bookkeeper-zkServers) is set to the connection string for the ZooKeeper cluster. The following is an example:
+You can configure BookKeeper bookies using the [`conf/bookkeeper.conf`](reference-configuration.md#bookkeeper) configuration file. The most important step in configuring bookies for our purposes here is ensuring that `metadataServiceUri` is set to the URI for the ZooKeeper cluster. The following is an example:
 
 ```properties
 
-zkServers=zk1.us-west.example.com:2181,zk2.us-west.example.com:2181,zk3.us-west.example.com:2181
+metadataServiceUri=zk://zk1.us-west.example.com:2181/ledgers
```

Comment:
I think `/ledgers` is not required?

Reply:
If I do the same connect string as in the `initialize-cluster-metadata` command - `zk:zookeeper:2181` - I get:

```
2022-06-22T10:22:37,902+0000 [main] ERROR org.apache.bookkeeper.server.Main - Failed to build bookie server
org.apache.bookkeeper.bookie.BookieException$MetadataStoreException: Failed to initialize metadata bookie driver
...
Caused by: org.apache.bookkeeper.meta.exceptions.MetadataException: java.lang.IllegalArgumentException: Invalid metadata service URI format: zk:zookeeper:2181
...
Caused by: java.lang.IllegalArgumentException: Invalid metadata service URI format: zk:zookeeper:2181
...
```

If I use a URI but omit the `/ledgers` path - `zk://zookeeper:2181` - I get:

```
[main] ERROR org.apache.bookkeeper.server.Main - Failed to build bookie server
java.lang.IllegalArgumentException: Path length must be > 0
...
```

Only `zk://zookeeper:2181/ledgers` is working for me